examples/bigquery-audit-log/bigquery_audit_log.sql

```diff
@@ -170,9 +172,12 @@ SELECT
   ((totalBilledBytes / 1000000000) / 1000) * 5 AS estimatedCostUsd,
   billingTier,
   query,
+  CONCAT(query.destinationTable.datasetId, '.', query.destinationTable.tableId) AS query_destinationTable_relativePath,
```

Comment:
Since the other columns are camel-case, this should probably be `queryDestinationTableRelativePath`

Reply:
Done